clients/fides-js/src/components/Overlay.tsx

```diff
@@ -107,13 +131,17 @@ const Overlay: FunctionComponent<OverlayProps> = ({
 
   return (
     <div id="fides-js-root">
-      <ConsentBanner
-        experience={experience.experience_config}
-        onAcceptAll={onAcceptAll}
-        onRejectAll={onRejectAll}
-        waitBeforeShow={100}
-        onOpenModal={() => setModalIsOpen(true)}
-      />
+      {experience.show_banner ? (
```

Comment:
This is an unrelated change, right?

Reply:
Its related. Since we now need to initialize the Overlay for the following use case:

* experience config `show_banner` is `false`
* experience is otherwise valid and has notices
* link el exists in DOM

I had to move this check for `show_banner` within the Overlay component itself